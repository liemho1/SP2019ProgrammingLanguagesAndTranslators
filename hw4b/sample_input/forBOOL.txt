{
  x = 1;
  for (x in FALSE)
    print(x);
  print("bad example")
}
