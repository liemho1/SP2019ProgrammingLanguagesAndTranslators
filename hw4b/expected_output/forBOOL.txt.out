Line 3: Arg 2 must be list
